src/components/Display.js

```diff
@@ -0,0 +1,122 @@
+          <div className="card" key={book.id}>
+            <div className="card-body">
+              <div className="container">
+                <div className="row">
+                  <div className="col">
+                    <p className="fs-6 text-secondary">{book.category}</p>
+                    <p className="fs-5 fw-bold">{book.title}</p>
+                    <p className="fs-5" id="typeface">{book.author}</p>
+                    <div className="container p-0">
+                      <div className="row" id="typeface">
+                        <div className="col">
+                          Comments
+                        </div>
+                        <div className="col">
+                          <button type="button" onClick={removeBookBtn} id={book.id} className="buttonRemove">Remove</button>
+                        </div>
+                        <div className="col">
+                          Edit
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div className="col">
+                    <div className="progress-circle">
+                      <div id="progress-circle">
+                        <CircularProgressbar value={randomNum} style={barstyle} />
+                      </div>
+                      <div>
+                        {randomNum}
+                        % Completed
+                      </div>
+                    </div>
+
+                  </div>
+                  <div className="col">
+                    <p>Current chapter</p>
+                    <p>
+                      Chapter&nbsp;
+
+                      {Math.floor(Math.random() * 100)}
+                    </p>
+                    <button type="button" className="btn btn-primary">UPDATE PROGRESS</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        );
+      })}
+    </div>
+  );
+
+  return (
+    <div className="container align-middle pt-5">
+      <div className="container my-5 align-middle" id="formContainer">
+        <div className="card">
+          <div className="card-header">
+            <div className="container">
+              <div className="row">
+                <div className="col-sm-4">
+                  <span className="Bookstore-CMS Text-Style-5">Bookstore CMS</span>
+                </div>
+                <div className="col-sm-2">
+                  <Link className="links" to="/">
+                    <span className="BOOKS Text-Style-7">
+                      BOOKS
+                    </span>
+                  </Link>
+                </div>
+                <div className="col-sm-2">
+                  <Link className="links" to="/categories">
+                    <span className="CATEGORIES Text-Style-7">
+                      CATEGORIES
+                    </span>
+                  </Link>
+                </div>
+              </div>
```

Comment:
- [ ] Great job implementiing thr navbar as requested, however, the background is still different from the one given.
- [ ] You will also notice that you are missing a user icon towards the right of the navbar. Kindly add it.
Yours
![image](https://user-images.githubusercontent.com/68167810/132100230-abc1f0b6-13af-4025-ba51-409c68da8491.png)
Given design:
![image](https://user-images.githubusercontent.com/68167810/132100253-4f35e4a9-41bf-455d-bd4f-1abee33cac17.png)

Reply:
Which background are you referring to?

Reply:
the background of the nav bar